src/commands/lambda/__tests__/instrument.test.ts

```diff
@@ -1416,39 +1252,37 @@ describe('lambda', () => {
             .rejects('Unexpected error updating request')
         }
 
-        const cli = makeCli()
-        const context = createMockContext()
-        const code = await cli.run(
-          [
-            'lambda',
-            'instrument',
-            '-f',
-            'arn:aws:lambda:us-east-1:123456789012:function:lambda-1-us-east-1',
-            '-f',
-            'arn:aws:lambda:us-east-1:123456789012:function:lambda-2-us-east-1',
-            '-f',
-            'arn:aws:lambda:us-east-1:123456789012:function:lambda-3-us-east-1',
-            '-f',
-            'arn:aws:lambda:us-east-2:123456789012:function:lambda-1-us-east-2',
-            '-f',
-            'arn:aws:lambda:us-east-2:123456789012:function:lambda-2-us-east-2',
-          ],
-          context
-        )
-        const output = context.stdout.toString()
+        const {code, context} = await runCLI([
+          '-f',
+          'arn:aws:lambda:us-east-1:123456789012:function:lambda-1-us-east-1',
+          '-f',
+          'arn:aws:lambda:us-east-1:123456789012:function:lambda-2-us-east-1',
+          '-f',
+          'arn:aws:lambda:us-east-1:123456789012:function:lambda-3-us-east-1',
+          '-f',
+          'arn:aws:lambda:us-east-2:123456789012:function:lambda-1-us-east-2',
+          '-f',
+          'arn:aws:lambda:us-east-2:123456789012:function:lambda-2-us-east-2',
+          '-f',
+          'arn:aws:lambda:us-east-2:123456789012:function:lambda-3-us-east-2',
+        ])
         expect(code).toBe(0)
-        expect(output).toMatchSnapshot()
+        expect(context.stdout.toString()).toMatchSnapshot()
       })
 
       test('aborts when every lambda function fails to update on instrument', async () => {
         ;(fs.readFile as any).mockImplementation((a: any, b: any, callback: any) => callback({code: 'ENOENT'}))
+        const failingLambdas = [
+          'arn:aws:lambda:us-east-1:123456789012:function:lambda-1-us-east-1',
+          'arn:aws:lambda:us-east-2:123456789012:function:lambda-1-us-east-2',
+        ]
         mockLambdaConfigurations(lambdaClientMock, {
           'arn:aws:lambda:us-east-1:123456789012:function:lambda-1-us-east-1': {
             config: {
               FunctionArn: 'arn:aws:lambda:us-east-1:123456789012:function:lambda-1-us-east-1',
               FunctionName: 'lambda-1-us-east-1',
               Handler: 'index.handler',
-              Runtime: 'nodejs22.x',
+              Runtime: 'nodejs16.x',
```

Comment:
Result of this move:https://github.com/DataDog/datadog-ci/pull/1622#discussion_r2053859312

(including the few lines below as well)